text/0015-market-design-revisit.md

```diff
@@ -0,0 +1,116 @@
+- It reverses the order of the market and renewal phase. 
+    - This allows to fine-tune the price through market forces.
+- It exposes the renewal prices, while still being beneficial for longterm tenants, more to market forces. 
+- It removes the LeadIn period and introduces a (from the perspective of the coretime systemchain) passive Settlement Phase, that allows the secondary market to exert it's force.
+
+The premise of this proposal is to reduce complexity by introducing a common price (that develops releative to capacity consumption of Polkadot UC), while still allowing for market forces to add efficiency. Longterm lease owners still receive priority **IF** they can pay (close to) the market price. This prevents a situation where the renewal price significantly diverges from renewal prices which allows for core captures. While maximum price increase certainty might seem contradictory to efficient price discovery, the proposed model aims to balance these elements, utilizing market forces to determine the price and allocate cores effectively within certain bounds. It must be stated, that potential price increases remain predictable (in the worst-case) but could be higher than in the originally proposed design. The argument remains, however, that we need to allow market forces to affect all prices for an efficient Coretime pricing and allocation.
+
+Ultimately, this the framework proposed here adheres to all requirements stated in RFC-1.
+
+## Stakeholders
+
+Primary stakeholder sets are:
+
+- Protocol researchers and developers, largely represented by the Polkadot Fellowship and Parity Technologies' Engineering division.
+- Polkadot Parachain teams both present and future, and their users.
+- Polkadot DOT token holders.
+
+## Explanation
+
+### Bulk Markets
+ 
+The `BULK_PERIOD` has been restructured into two primary segments: the `MARKET_PERIOD` and `RENEWAL_PERIOD`, along with an auxiliary `SETTLEMENT_PERIOD`. This latter period doesn't necessitate any actions from the coretime system chain, but it facilitates a more efficient allocation of coretime in secondary markets. A significant departure from the original proposal lies in the timing of renewals, which now occur post-market phase. This adjustment aims to harmonize renewal prices with their market counterparts, ensuring a more consistent and equitable pricing model.
+
+#### Market Period (14 days)
+
+During the market period, core sales are conducted through a well-established **clearing price Dutch auction** that features a `RESERVE_PRICE`. The price initiates at a premium, designated as `PRICE_PREMIUM` (for instance, 30%) and descends linearly to the `RESERVE_PRICE` throughout the duration of the `MARKET_PERIOD`. Each bidder is expected to submit both their desired price and the quantity (that is, the amount of Coretime) they wish to purchase. To secure these acquisitions, bidders must make a deposit equivalent to their bid multiplied by the chosen quantity, in DOT. 
```

Comment:
Unfortunately (and this wasn't stated absolutely clearly in RFC-1), dutch auctions are quite difficult to implement correctly in a blockchain environment with bounded storage and computation costs. The "lead-in" period approach of RFC-1 trades off theoretical elegance for implementation elegance.

I tend to agree that auctions for bulk coretime (followed by splitting and trading) should be the natural conclusion for the market mechanism, but it is too early for that.

Reply:
I cannot judge whether the stated mechanism exceeds the available storage and computation costs, so I'd leave that assessment to experts like you.